examples/client/lwm2mclient.c

```diff
@@ -844,9 +938,14 @@ int main(int argc, char *argv[])
     lwm2m_client_state_t previousState = STATE_INITIAL;
 #endif
 
+
+#if 0 
+#else
```

Comment:
I turned on and off DTLS PSK with this. Of course my key material has been deleted.

From: David Navarro [mailto:notifications@github.com]
Sent: Monday, April 25, 2016 6:29 PM
To: eclipse/wakaama <wakaama@noreply.github.com>
Cc: Frank Gerlach <Frank_Gerlach@epam.com>; Author <author@noreply.github.com>
Subject: Re: [eclipse/wakaama] improved the ASCII hex to binary key conversion. (#147)


In examples/client/lwm2mclient.c<https://github.com/eclipse/wakaama/pull/147#discussion_r60933443>:

> @@ -844,9 +938,14 @@ int main(int argc, char *argv[])

>      lwm2m_client_state_t previousState = STATE_INITIAL;

>  #endif

>

> +

> +#if 0

> +#else

what is the need for this ?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/eclipse/wakaama/pull/147/files/0a97caab6faaf3616adacba0a0a8d15183f1a6ad#r60933443>